docs/software/loan-model.md

```diff
@@ -0,0 +1,77 @@
+---
+template: main.html
+---
+
+<img src="https://raw.githubusercontent.com/ExpressLRS/ExpressLRS-Hardware/master/img/software.png">
+
+## Loan Model
+
+!!! note "Scenario"
+    You are flying with by buddies and your friend has crashed all his models. You'd like to be able to lend him one of yours but you
+    can't because it's using a binding phrase, so he's relegated to being the crached model fetcher for the rest of the day!
+
+    No longer is he destined to be running to the bando and back again fetching downed quads. You can use the "Loan Model" feature and lend him one of your models!
+
+## How to "Loan Model"
+
+### On the lenders handset
+
+1. Connect to the model you wish to loan out with your handset
+2. Enter the LUA screen and select "Other Devices" at the bottom of the list
+
+    ![Other Devices](../assets/images/loan-other-devices.png)
+
+3. Choose the RX from the list
+
+    ![Select RX](../assets/images/loan-rx-select.png)
+
+4. Select the "Loan Model" entry
+
+    ![Loan Model](../assets/images/loan-rx-menu.png)
+
+5. The receiver LED should go into the bind flashing state
+
+    ![LEDSEQ_BINDING](https://cdn.discordapp.com/attachments/738450139693449258/921065812763218010/LEDSEQ_BINDING_10_10_10_100.gif)
+
+### On the borrowers handset
+1. From the main ExpressLRS LUA screen select `[Bind]`
+2. The receiver LED should turn solid on, and the handset should connect to the recevier
+    ![CONNECTED](https://cdn.discordapp.com/attachments/738450139693449258/921065812507373568/LED_ON.gif)
+
+## Returning a "Loaned" Model
+
+There are a couple of ways to return a model to the owner. 
+
+The first method is for the borrower to return it from the LUA script as follows:
+
+  1. On the borrowers handset, connect to the "loaned" model
+  2. Enter the LUA screen and select "Other Devices" at the bottom of the list
+
+      ![Other Devices](../assets/images/loan-other-devices.png)
+
+  3. Choose the RX from the list
+
+      ![Select RX](../assets/images/loan-rx-select.png)
+
+  4. Select the "Loan Model" entry
+
+      ![Return Model](../assets/images/loan-return.png)
+
+The second method is as follows, used if the borrower forgets to return the model:
```

Comment:
The second method, used if the borrower forgets to return the model, is as follows:

(just a suggestion. No strong inclination.)